### ISSUE TITLE
pretrain process code

### ISSUE THREAD

**Issue by xhl-xhl** [TencentAILabHealthcare/scBERT#67]
with torch.no_grad():
    for index, data in enumerate(val_loader):
        index += 1
        data = data.to(device)
        data, labels = data_mask(data)
        logits = model(data)
        loss = loss_fn(logits.transpose(1, 2), labels)
        running_loss += loss.item()
        softmax = nn.Softmax(dim=-1)
        final = softmax(logits)[..., 1:-1]
        final = final.argmax(dim=-1) + 1
        predictions.append(final)
        truths.append(labels)
Can you explain why softmax only operates on 5 bin（softmax(logits)[..., 1:-1]），however model.to_out bin is 7(ground true lable) during pre-training. Thanks.
